definitions/contested/json/CaseField/CaseField.json

```diff
@@ -2641,5 +2641,14 @@
     "HintText": "Please add any additional comments for  court admin (this comments will not be accesible by applicant's solicitor)",
     "FieldType": "Text",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "Contested_ConsentedApprovedOrders",
+    "Label": "Consent Order Documents",
```

Comment:
might not be part of this story but it might be worth adding this field to the consent tab while you're capturing it here